src/content/transformers/promptInputFieldsTransformer.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  Transformer,
+  PromptInputFieldsAugmented,
+  PromptInputFieldsRaw,
+  PromptInputQuestion
+} from '../content.types';
+
+export const promptInputFieldsTransformer: Transformer<
+  PromptInputFieldsRaw,
+  PromptInputFieldsAugmented
+> = (prompts: PromptInputFieldsRaw[]): PromptInputFieldsAugmented[] => {
+  const promptInputFields = {};
+  for (const prompt of prompts) {
+    const entry: PromptInputQuestion[] = [];
+    const key = prompt.data.fieldId;
+    for (const input of prompt.data.contentInput) {
+      const { title, helperText } = input.fields;
+      entry.push({
+        title,
+        helperText
+      })
+    }
+    promptInputFields[key] = entry;
+  }
+
+  return [promptInputFields];
```

Comment:
You can accomplish this logic w/ `.reduce` and `.map`

Reply:
You're really trying to make me a FP bro huh, haha. Changed it

Reply:
FYI.. it's a good habit to not be mutating the properties of function parameters, even in `.reduce`...

if you ever do end up doing a reduce, and want to have some object/array as a starting point for the `acc`.. you could end up implicitly mutating the original variable as you go through the reduce...

![image](https://github.com/user-attachments/assets/b395cb45-7db3-4f1f-bdce-1484c623b42d)

const result = arr.reduce((acc, {name}) => ({...acc, [name]: name}))